cmd/admin-handlers.go

```diff
@@ -440,6 +444,33 @@ func (a adminAPIHandlers) HealHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (a adminAPIHandlers) getConfigDotJSON() ([]byte, error) {
+	// Take a read lock on minio/config.json. NB minio is a
```

Comment:
What does `NB` stand for? Is it `NS`?

Reply:
> What does NB stand for? Is it NS?

NB means note well, I am not sure what you are used to write

Reply:
You can change it to `Note that`